src/onINP.ts

```diff
@@ -162,31 +245,21 @@ export const onINP = (onReport: INPReportCallback, opts?: ReportOpts) => {
 
     const handleEntries = (entries: INPMetric['entries']) => {
       entries.forEach((entry) => {
-        if (entry.interactionId) {
-          processEntry(entry);
+        // Skip `first-input` entries if any `event` entries have already
```

Comment:
I haven't audited that carefully-- but we could have a case where some event timings (like hover/pointerover) may already be reported, but then we get our first discrete input.  It might still be <40ms and not be reported as and `event` but be reported as `first-input`.  So `first-input` is still valuable sometimes as a specicial event timing...

Reply:
Ahh, yes, good call. I can go back to checking startTime/duration, but I'll still move this check to the top of this function so it doesn't get called twice in cases where the `first-input` entry *does* have an interaction ID.

Reply:
Resolving this as I've updated the logic to handle `first-input` entries.